pkg/cache/cluster_test.go

```diff
@@ -26,7 +26,7 @@ import (
 func strToUnstructured(jsonStr string) *unstructured.Unstructured {
 	obj := make(map[string]interface{})
 	err := yaml.Unmarshal([]byte(jsonStr), &obj)
-	errors.CheckError(err)
+	errors.CheckErrorWithCode(err, errors.ErrorCommandSpecific)
```

Comment:
I think the tests can just exit with any exit code and do not need adaption. They should just use `CheckError()` or even better, `panic()` (see below).